GLideN64/.gitignore

```diff
@@ -19,5 +19,6 @@ src/Revision.h
 /projects/cmake/Makefile
 /projects/cmake/cmake_install.cmake
 /projects/cmake/osal/
+/projects/cmake/mupen64plus-video-GLideN64_autogen/
```

Comment:
I wasn't sure whether modifying this directory would cause later merge conflicts. From what I could tell, the GLideN64 folder is not automatically brought over, right? This new folder to ignore might be osx-specific

Reply:
Yeah this shouldn't go here, but it's not a big deal, it'll get overwritten the next time the pull_repos.sh. pull_repos.sh updates all the subprojects automatically.

m64p pulls GLideN64 from this branch: https://github.com/loganmc10/GLideN64/tree/NUI, which is upstream GLideN64 plus a few modifications.

I submitted a PR to GLideN64 to have the gitignore stuff upstreamed: https://github.com/gonetz/GLideN64/pull/2366